Assets/Mirror/Runtime/InterestManagement/InterestManagement.cs

```diff
@@ -0,0 +1,133 @@
+﻿// interest management from DOTSNET
+using UnityEngine;
+
+namespace Mirror
+{
+    [DisallowMultipleComponent]
+    [RequireComponent(typeof(NetworkManager))]
```

Comment:
why does this require `NetworkManager` ? doesn't look like NetworkManager is used anywhere in this script